kornia/geometry/transform/elastic_transform.py

```diff
@@ -14,50 +15,70 @@ def elastic_transform2d(
     noise: torch.Tensor,
     kernel_size: Tuple[int, int] = (63, 63),
     sigma: Tuple[float, float] = (32., 32.),
-    alpha: Tuple[float, float] = (1., 1.),
+    alpha: Tuple[float, float] = (16., 16.),
     align_corners: bool = False,
-    mode: str = 'bilinear'
+    mode: str = 'bilinear',
+    approach: str = 'coarse_noise'
 ) -> torch.Tensor:
     r"""Applies elastic transform of images as described in :cite:`Simard2003BestPF`.
 
+    .. warning::
+        'smoothing' approach is deprecated and will be removed > 0.6.0. Pleas use the more computational efficent
```

Comment:
I would change the warning from " will be removed > 0.6.0." to "
 Please use the more computationally efficient  (500x) times approach "upsampling"